compiler/one-cmds/CMakeLists.txt

```diff
@@ -8,6 +10,7 @@ set(ONE_COMMAND_FILES
     one-pack
     one-codegen
     one-prepare-venv
+    one_utils.py
```

Comment:
Adding files in `ONE_COMMAND_FILES` will make it executable. Please add another list for `one-build.cfg` and `one_utils.py`.